frontend/src/pages/CourseEditor/CourseEditorContainer.tsx

```diff
@@ -8,7 +8,13 @@ import CourseEditor from './CourseEditor';
 
 const CourseEditorContainer: React.FC = () => {
   const formik = useFormik({
-    initialValues: {},
+    initialValues: {
+      videoURL: '',
+      textDescription: '',
+      exerciseTitle: '',
+      exerciseDescription: '',
+    },
```

Comment:
I guess, here should be course materials instead of empty strings in case of editing